[2024-10-03T19:52:55.557+0530] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-03T19:52:55.567+0530] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_ingestion_dag.validate_data manual__2024-10-03T14:22:41.727449+00:00 [queued]>
[2024-10-03T19:52:55.573+0530] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_ingestion_dag.validate_data manual__2024-10-03T14:22:41.727449+00:00 [queued]>
[2024-10-03T19:52:55.573+0530] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-10-03T19:52:55.600+0530] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): validate_data> on 2024-10-03 14:22:41.727449+00:00
[2024-10-03T19:52:55.610+0530] {standard_task_runner.py:72} INFO - Started process 22619 to run task
[2024-10-03T19:52:55.617+0530] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'data_ingestion_dag', 'validate_data', 'manual__2024-10-03T14:22:41.727449+00:00', '--job-id', '106', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_dag.py', '--cfg-path', '/tmp/tmpwcuz4gru']
[2024-10-03T19:52:55.622+0530] {standard_task_runner.py:105} INFO - Job 106: Subtask validate_data
[2024-10-03T19:52:55.681+0530] {task_command.py:467} INFO - Running <TaskInstance: data_ingestion_dag.validate_data manual__2024-10-03T14:22:41.727449+00:00 [running]> on host Nithesh-Kumar.localdomain
[2024-10-03T19:52:55.770+0530] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='data_ingestion_dag' AIRFLOW_CTX_TASK_ID='validate_data' AIRFLOW_CTX_EXECUTION_DATE='2024-10-03T14:22:41.727449+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-03T14:22:41.727449+00:00'
[2024-10-03T19:52:55.771+0530] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-03T19:52:55.846+0530] {file_data_context.py:222} INFO - FileDataContext loading fluent config
[2024-10-03T19:52:55.849+0530] {config.py:187} INFO - Loading 'datasources' ->
[]
[2024-10-03T19:52:55.944+0530] {validator.py:1467} INFO - 	28 expectation(s) included in expectation_suite.
[2024-10-03T19:52:56.200+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   0%|          | 0/73 [00:00<?, ?it/s]
[2024-10-03T19:52:56.200+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   0%|          | 0/73 [00:00<?, ?it/s]
[2024-10-03T19:52:56.201+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   3%|2         | 2/73 [00:00<00:00, 2227.46it/s]
[2024-10-03T19:52:56.242+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   3%|2         | 2/73 [00:00<00:01, 47.63it/s]  
[2024-10-03T19:52:56.244+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   4%|4         | 3/73 [00:00<00:01, 69.43it/s]
[2024-10-03T19:52:56.290+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   4%|4         | 3/73 [00:00<00:02, 33.38it/s]
[2024-10-03T19:52:56.304+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:  23%|##3       | 17/73 [00:00<00:00, 164.45it/s]
[2024-10-03T19:52:56.304+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:  23%|##3       | 17/73 [00:00<00:00, 164.45it/s]
[2024-10-03T19:52:56.338+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:  23%|##3       | 17/73 [00:00<00:00, 164.45it/s]
[2024-10-03T19:52:56.372+0530] {logging_mixin.py:190} WARNING - Calculating Metrics: 100%|##########| 73/73 [00:00<00:00, 164.45it/s]
[2024-10-03T19:52:56.388+0530] {logging_mixin.py:190} WARNING - Calculating Metrics: 100%|##########| 73/73 [00:00<00:00, 164.45it/s]
[2024-10-03T19:52:56.389+0530] {logging_mixin.py:190} WARNING - Calculating Metrics: 100%|##########| 73/73 [00:00<00:00, 164.45it/s]
[2024-10-03T19:52:56.389+0530] {logging_mixin.py:190} WARNING - Calculating Metrics: 100%|##########| 73/73 [00:00<00:00, 386.55it/s]
[2024-10-03T19:52:56.754+0530] {logging_mixin.py:190} INFO - Data quality check passed.
[2024-10-03T19:52:57.476+0530] {python.py:240} INFO - Done. Returned value was: None
[2024-10-03T19:52:57.482+0530] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-03T19:52:57.482+0530] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=data_ingestion_dag, task_id=validate_data, run_id=manual__2024-10-03T14:22:41.727449+00:00, execution_date=20241003T142241, start_date=20241003T142255, end_date=20241003T142257
[2024-10-03T19:52:57.521+0530] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-03T19:52:57.545+0530] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-10-03T19:52:57.559+0530] {local_task_job_runner.py:245} INFO - ::endgroup::
